src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - Avoid using `import` in non-root components. As per our checklist, you should prepare your data in one place (App component) and pass it to child components. This will make your components more reusable and easier to test.